integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -700,6 +726,16 @@ public void ttlRenameTest() throws Exception {
     Assert.assertEquals(ttl, folderInfo.ttl);
   }
 
+  private void setAuthenticateUserforConcurrentOperation(String username) {
```

Comment:
Right. Deleted it, and moved this method's comment along with field `TEST_AUTHENTICATE_USER`